.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Description
+## What is the current behavior?
+<!-- Please describe the current behavior that you are modifying. -->
+
+
+<!-- Issues are required for both bug fixes and features. -->
+Issue URL:
+
+
+## What is the new behavior?
+<!-- Please describe the behavior or changes that are being added by this PR. -->
+
+-
+-
+-
+
+## Does this introduce a breaking change?
+
+- [ ] Yes
+- [ ] No
```

Comment:
Perhaps we should represent this without using a check-box, otherwise, Github considers there is a pending task within the issue:
![Screenshot from 2022-06-22 13-44-03](https://user-images.githubusercontent.com/272048/175031951-db7dbef4-d7cb-4ee9-a7c8-d1c739dac600.png)

Reply:
Ahhh good catch there